### DISCUSSION TITLE
HM-1500 openDTU v24.4.12 |  Nothing received, resend whole request

### DISCUSSION THREAD

**Discussion by ds2k5** [tbnobody/OpenDTU#1929] [General]
Hello,
using a esp32 with NRF2401+ and openDTU v24.4.12.
The inverter will not talk with openDTU.
See no data about the generated electricity.

The openDTU nrf transmit power is set to:  high
the openDTU is around 3 meters away from the inverter outside
because I tryed in house frist... then outside
but did not connect...

Did reset inverter and openDTU too...

See this in console:

```
18:23:49.247 > Admin AP remaining seconds: 150 / 180

18:23:49.310 > RX Period End

18:23:49.310 > All missing

18:23:49.310 > Nothing received, resend whole request

18:23:49.310 > TX RealTimeRunData Channel: 3 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 12 00 00 00 00 00 00 00 00 0F 9D 24

18:23:49.815 > RX Period End

18:23:49.815 > All missing

18:23:49.815 > Nothing received, resend count exeeded

18:23:50.170 > TX AlarmData Channel: 23 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 12 00 00 00 00 00 00 00 00 D5 86 FF

18:23:50.788 > RX Period End

18:23:50.788 > All missing

18:23:50.788 > Nothing received, resend whole request

18:23:50.788 > TX AlarmData Channel: 40 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 12 00 00 00 00 00 00 00 00 D5 86 FF

18:23:51.637 > RX Period End

18:23:51.637 > All missing

18:23:51.637 > Nothing received, resend whole request

18:23:51.637 > TX AlarmData Channel: 61 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 12 00 00 00 00 00 00 00 00 D5 86 FF

18:23:52.499 > RX Period End

18:23:52.499 > All missing

18:23:52.499 > Nothing received, resend whole request

18:23:52.499 > TX AlarmData Channel: 75 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 12 00 00 00 00 00 00 00 00 D5 86 FF

18:23:53.355 > RX Period End

18:23:53.355 > All missing

18:23:53.355 > Nothing received, resend whole request

18:23:53.355 > TX AlarmData Channel: 3 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 12 00 00 00 00 00 00 00 00 D5 86 FF

18:23:54.117 > RX Period End

18:23:54.117 > All missing

18:23:54.117 > Nothing received, resend count exeeded

18:23:54.117 > Fetch inverter: 116190757598

18:23:54.496 > TX RealTimeRunData Channel: 23 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 1A 00 00 00 00 00 00 00 00 CF FA 8B

18:23:54.827 > RX Period End

18:23:54.827 > All missing

18:23:54.827 > Nothing received, resend whole request

18:23:54.827 > TX RealTimeRunData Channel: 40 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 1A 00 00 00 00 00 00 00 00 CF FA 8B

18:23:55.448 > RX Period End

18:23:55.448 > All missing

18:23:55.448 > Nothing received, resend whole request

18:23:55.448 > TX RealTimeRunData Channel: 61 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 1A 00 00 00 00 00 00 00 00 CF FA 8B

18:23:56.066 > RX Period End

18:23:56.066 > All missing

18:23:56.066 > Nothing received, resend whole request

18:23:56.066 > TX RealTimeRunData Channel: 75 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 1A 00 00 00 00 00 00 00 00 CF FA 8B

18:23:56.679 > RX Period End

18:23:56.679 > All missing

18:23:56.679 > Nothing received, resend whole request

18:23:56.679 > TX RealTimeRunData Channel: 3 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 1A 00 00 00 00 00 00 00 00 CF FA 8B

18:23:57.178 > RX Period End

18:23:57.178 > All missing

18:23:57.178 > Nothing received, resend count exeeded

18:23:57.282 > TX AlarmData Channel: 23 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 1A 00 00 00 00 00 00 00 00 15 E1 50

18:23:58.143 > RX Period End

18:23:58.143 > All missing

18:23:58.143 > Nothing received, resend whole request

18:23:58.143 > TX AlarmData Channel: 40 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 1A 00 00 00 00 00 00 00 00 15 E1 50

18:23:59.005 > RX Period End

18:23:59.005 > All missing

18:23:59.005 > Nothing received, resend whole request

18:23:59.005 > TX AlarmData Channel: 61 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 1A 00 00 00 00 00 00 00 00 15 E1 50

18:23:59.289 > Admin AP remaining seconds: 160 / 180

18:23:59.862 > RX Period End

18:23:59.862 > All missing

18:23:59.862 > Nothing received, resend whole request

18:23:59.862 > TX AlarmData Channel: 75 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 1A 00 00 00 00 00 00 00 00 15 E1 50

18:24:00.730 > RX Period End

18:24:00.730 > All missing

18:24:00.730 > Nothing received, resend whole request

18:24:00.730 > TX AlarmData Channel: 3 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 1A 00 00 00 00 00 00 00 00 15 E1 50

18:24:01.476 > RX Period End

18:24:01.476 > All missing

18:24:01.476 > Nothing received, resend count exeeded

18:24:01.476 > Fetch inverter: 116190757598

18:24:01.583 > TX RealTimeRunData Channel: 23 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 21 00 00 00 00 00 00 00 00 FE DD A6

18:24:02.199 > RX Period End

18:24:02.199 > All missing

18:24:02.199 > Nothing received, resend whole request

18:24:02.199 > TX RealTimeRunData Channel: 40 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 21 00 00 00 00 00 00 00 00 FE DD A6

18:24:02.808 > RX Period End

18:24:02.808 > All missing

18:24:02.808 > Nothing received, resend whole request

18:24:02.808 > TX RealTimeRunData Channel: 61 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 21 00 00 00 00 00 00 00 00 FE DD A6

18:24:03.416 > RX Period End

18:24:03.416 > All missing

18:24:03.416 > Nothing received, resend whole request

18:24:03.416 > TX RealTimeRunData Channel: 75 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 21 00 00 00 00 00 00 00 00 FE DD A6

18:24:04.030 > RX Period End

18:24:04.030 > All missing

18:24:04.030 > Nothing received, resend whole request

18:24:04.030 > TX RealTimeRunData Channel: 3 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 21 00 00 00 00 00 00 00 00 FE DD A6

18:24:04.530 > RX Period End

18:24:04.530 > All missing

18:24:04.530 > Nothing received, resend count exeeded

18:24:04.643 > TX AlarmData Channel: 23 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 21 00 00 00 00 00 00 00 00 24 C6 7D

18:24:05.498 > RX Period End

18:24:05.498 > All missing

18:24:05.498 > Nothing received, resend whole request

18:24:05.498 > TX AlarmData Channel: 40 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 21 00 00 00 00 00 00 00 00 24 C6 7D

18:24:06.354 > RX Period End

18:24:06.354 > All missing

18:24:06.354 > Nothing received, resend whole request

18:24:06.354 > TX AlarmData Channel: 61 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 21 00 00 00 00 00 00 00 00 24 C6 7D

18:24:07.211 > RX Period End

18:24:07.211 > All missing

18:24:07.211 > Nothing received, resend whole request

18:24:07.211 > TX AlarmData Channel: 75 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 21 00 00 00 00 00 00 00 00 24 C6 7D

18:24:08.075 > RX Period End

18:24:08.075 > All missing

18:24:08.075 > Nothing received, resend whole request

18:24:08.075 > TX AlarmData Channel: 3 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 21 00 00 00 00 00 00 00 00 24 C6 7D

18:24:08.829 > RX Period End

18:24:08.829 > All missing

18:24:08.829 > Nothing received, resend count exeeded

18:24:08.829 > Fetch inverter: 116190757598

18:24:08.935 > TX RealTimeRunData Channel: 23 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 28 00 00 00 00 00 00 00 00 AE B7 95

18:24:09.366 > Admin AP remaining seconds: 170 / 180

18:24:09.550 > RX Period End

18:24:09.550 > All missing

18:24:09.550 > Nothing received, resend whole request

18:24:09.550 > TX RealTimeRunData Channel: 40 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 28 00 00 00 00 00 00 00 00 AE B7 95

18:24:10.157 > RX Period End

18:24:10.157 > All missing

18:24:10.157 > Nothing received, resend whole request

18:24:10.157 > TX RealTimeRunData Channel: 61 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 28 00 00 00 00 00 00 00 00 AE B7 95

18:24:10.763 > RX Period End

18:24:10.763 > All missing

18:24:10.763 > Nothing received, resend whole request

18:24:10.763 > TX RealTimeRunData Channel: 75 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 28 00 00 00 00 00 00 00 00 AE B7 95

18:24:11.376 > RX Period End

18:24:11.376 > All missing

18:24:11.376 > Nothing received, resend whole request

18:24:11.376 > TX RealTimeRunData Channel: 3 --> 15 90 75 75 98 80 11 18 12 80 0B 00 66 23 EC 28 00 00 00 00 00 00 00 00 AE B7 95

18:24:11.878 > RX Period End

18:24:11.878 > All missing

18:24:11.878 > Nothing received, resend count exeeded

18:24:11.994 > TX AlarmData Channel: 23 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 28 00 00 00 00 00 00 00 00 74 AC 4E

18:24:12.842 > RX Period End

18:24:12.842 > All missing

18:24:12.842 > Nothing received, resend whole request

18:24:12.842 > TX AlarmData Channel: 40 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 28 00 00 00 00 00 00 00 00 74 AC 4E

18:24:13.710 > RX Period End

18:24:13.710 > All missing

18:24:13.710 > Nothing received, resend whole request

18:24:13.710 > TX AlarmData Channel: 61 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 28 00 00 00 00 00 00 00 00 74 AC 4E

18:24:14.574 > RX Period End

18:24:14.574 > All missing

18:24:14.574 > Nothing received, resend whole request

18:24:14.574 > TX AlarmData Channel: 75 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 28 00 00 00 00 00 00 00 00 74 AC 4E

18:24:15.433 > RX Period End

18:24:15.433 > All missing

18:24:15.433 > Nothing received, resend whole request

18:24:15.433 > TX AlarmData Channel: 3 --> 15 90 75 75 98 80 11 18 12 80 11 00 66 23 EC 28 00 00 00 00 00 00 00 00 74 AC 4E

 
```

I had tested the box with my HM-600
before I shipped to to my friend.

What did I wrong ?

**Comment by MetaChuh** [comment]
@ds2k5 

> I had tested the box with my HM-600
before I shipped to to my friend.

> What did I wrong ?

maybe you did nothing wrong.
it's difficult to say, as your assembled opendtu is not at your place anymore.

can you ask your friend, if he sees a green light flashing at the hm-1500 inverter ?
maybe he did not connect the panels correctly or he only connected the ac line.
the hm-1500 needs dc power to boot up.

greetings,
metachuh

**Reply by MetaChuh** [reply]
@Landorin-GH 

this is not an issue about hms inverters, as they use a different radio module than the hm inverters mentioned here.
this issue is at most about the nrf24+ module.

greetings,
metachuh

**Comment by ds2k5** [comment]
@MetaChuh 

>can you ask your friend, if he sees a green light flashing at the hm-1500 inverter ?

the LED is permanent green

@Landorin-GH

Had the hole package esp32 + nrf24+  testet with my HM-600
got data from my inverter.
So I delted my Inverter from config
and send the package to my friend
he added the serial number form his hm-1500

Could this be a powerSupply issue ?
the the usb power supply did not have enough power ?

**Reply by MetaChuh** [reply]
@ds2k5 

> the LED is permanent green

it should blink on an hm-1500 if everything is ok.

according to hoymiles, a static green led is not a listed option.

<img width="999" alt="Bildschirmfoto 2024-04-20 um 21 30 20" src="https://github.com/tbnobody/OpenDTU/assets/45773305/ae1d0a99-855a-4664-9863-5d9240ee9b31">

**Comment by ds2k5** [comment]
@MetaChuh 

thanks / danke

Will send to my friend / werde ich meinem freund schicken

**Comment by ds2k5** [comment]
@MetaChuh

he wrote me

green LED is blinking in 1 sec rythmus

**Reply by MetaChuh** [reply]
great, that sounds better.

do you have temporary remote access (teamviewer or similar) to him and the opendtu web interface ?
or could you tell him to check and send you a screenshot of the bottom part of `info > system` like shown here:

<img width="1358" alt="Bildschirmfoto 2024-04-21 um 14 03 07" src="https://github.com/tbnobody/OpenDTU/assets/45773305/462de610-ad84-4138-a783-d29d1437ba6a">

and please ask him to change the nrf24+ transmitting power down to minimum.

**Reply by ds2k5** [reply]
@MetaChuh 

yes this is all 3 points green as in you picture

![IMG_9946](https://github.com/tbnobody/OpenDTU/assets/11292286/a4e812b3-4c10-4595-b158-75d788580891)


He switched to minimum TX power
but did not change

![IMG_9945](https://github.com/tbnobody/OpenDTU/assets/11292286/888b9729-2ac8-481a-a83d-984cb5752232)


Log file:
[konsole.txt](https://github.com/tbnobody/OpenDTU/files/15066041/konsole.txt)

![IMG_9949](https://github.com/tbnobody/OpenDTU/assets/11292286/a15008f4-9891-448d-8907-fc575778ad12)

![IMG_9948](https://github.com/tbnobody/OpenDTU/assets/11292286/ff2c6ad0-9a15-405b-a1da-633fcc9e5157)

![IMG_9950](https://github.com/tbnobody/OpenDTU/assets/11292286/420f0c62-538c-40f8-8d48-acca96977ff7)




![IMG_9951](https://github.com/tbnobody/OpenDTU/assets/11292286/cbc0b38a-21a2-41da-b91b-3d307822c03d)



![IMG_9957](https://github.com/tbnobody/OpenDTU/assets/11292286/439c5753-4440-4f43-9f5c-feadb06bd2b2)


![IMG_9953](https://github.com/tbnobody/OpenDTU/assets/11292286/f24ec239-f0d4-4746-8ebc-7e6b403b64b9)

**Reply by MetaChuh** [reply]
eventuell doch sein inverter funkmodul, wenn die dtu bei dir funktioniert hat, oder antenne nicht korrekt aufgeschraubt, wenn abschraubbar ?

morgen noch bitte andere sendestärken probieren (angefangen mit low)

du könntest auch versuchen ihm morgen per fernwartung die [`v24.3.15`](https://github.com/tbnobody/OpenDTU/releases/tag/v24.3.15) via ota einzuspielen, um zu sehen ob dies einen unterschied macht.

sideinfo: derzeit sind bei mir zum testen: hm-1500, hm-800, hm-600, und hm-300, alle problemlos mit v24.4.12
die dauertests tests waren ebenfalls ohne probleme mit v24.3.15

**Reply by ds2k5** [reply]
@MetaChuh 

>eventuell doch sein inverter funkmodul, wenn die dtu bei dir funktioniert hat, oder antenne nicht korrekt aufgeschraubt, wenn >abschraubbar ?

ist ein Modul ohne Anschraubantenne

>du könntest auch versuchen ihm morgen per fernwartung die [v24.3.15](https://github.com/tbnobody/OpenDTU/releases
>/tag/v24.3.15) via ota einzuspielen, um zu sehen ob dies einen unterschied macht.

ich kanns versuchen wenn er zeit hat....
also diese:  https://github.com/tbnobody/OpenDTU/releases/download/v24.3.15/opendtu-generic_esp32.bin


Gibts eine Chance mit einer Smartphone APP / Laptop APP zu prüfen ob der Inverter ein aktives/funktionierendes Modul hat? Ohne extra Hardware

Zum anderen noch eine Frage muss man einen 100pF Kondensator am nRF24+ haben ?

**Reply by MetaChuh** [reply]
> ich kanns versuchen wenn er zeit hat....
also diese: https://github.com/tbnobody/OpenDTU/releases/download/v24.3.15/opendtu-generic_esp32.bin

ja genau.

> Gibts eine Chance mit einer Smartphone APP / Laptop APP zu prüfen ob der Inverter ein aktives/funktionierendes Modul hat? Ohne extra Hardware

soweit ich weiß nicht.

> muss man einen 100pF Kondensator am nRF24+ haben ?

immer empfehlenswert für die stabilität wegen spannungseinbrüche ausgleichen, jedoch sollte auch ohne zumindest zeitweise eine funkkommunikation stattfinden.

**Reply by ds2k5** [reply]
@MetaChuh  danke

@schneeer danke

**Reply by ds2k5** [reply]
@MetaChuh 

also auch mit der Version 24.3.15 verhält es sich gleich.

neues/anderes nRF24+ Modul versuchen ?
Müsste bestellen und zu Ihm schicken

**Reply by MetaChuh** [reply]
hi,
kann ich nicht sagen, vor allem weil dein funkmodul und die gesamte dtu ja bei dir funktioniert hat.

solche fälle sind extrem schwierig, weil du ja auch nicht um die ecke von ihm bist, und mal kurz mit deiner jetzigen dtu antanzen könntest, oder seinen inverter zu dir holst.

**Comment by ds2k5** [comment]
@MetaChuh 

verstehe danke

OK ich bau nen neue openDTU zusammen
und schick im diese

**Comment by ds2k5** [comment]
also die neue openDTU ist angekommen und läuft

**Reply by MetaChuh** [reply]
sehr gut 👍
danke dir für die rückmeldung.
